src/parser.js

```diff
@@ -1148,7 +1148,7 @@ Parser.prototype.tok = function() {
       while (this.next().type !== "list_item_end") {
         body.push(
           this.token.type === "text"
-            ? this.renderer.paragraph(this.inline.parse(this.token.text))
```

Comment:
What was the reason for this change? Is it necessary for Slate 0.47? It will likely cause issues per @tommoor's comments.